examples/typescript-server.ts

```diff
@@ -75,4 +75,5 @@ server.listen({ port: 8080 }, (err, address) => {
     console.error(err);
     process.exit(1);
   }
+   console.log(`server listening on ${address}`)
```

Comment:
Cant we use instead of console.log server.log.info?

Reply:
No, it does not exist on the server (which I guess why other logs are being logged through the console)